src/ks-clock-face.c

```diff
@@ -24,55 +24,57 @@ static bool s_animating = false;
 
 /*************************** AnimationImplementation **************************/
 
-static void animation_started(Animation *anim, void *context) {
+static void prv_animation_started(Animation *anim, void *context) {
   s_animating = true;
 }
 
-static void animation_stopped(Animation *anim, bool stopped, void *context) {
+static void prv_animation_stopped(Animation *anim, bool stopped, void *context) {
   s_animating = false;
 }
 
-static void animate(int duration, int delay, AnimationImplementation *implementation, bool handlers) {
+static void prv_animate(int duration, int delay, AnimationImplementation *implementation, bool handlers) {
   Animation *anim = animation_create();
   animation_set_duration(anim, duration);
   animation_set_delay(anim, delay);
   animation_set_curve(anim, AnimationCurveEaseInOut);
   animation_set_implementation(anim, implementation);
-  if(handlers) {
+  if (handlers) {
     animation_set_handlers(anim, (AnimationHandlers) {
-      .started = animation_started,
-      .stopped = animation_stopped
+      .started = prv_animation_started,
+      .stopped = prv_animation_stopped
     }, NULL);
   }
   animation_schedule(anim);
 }
 
 /************************************ UI **************************************/
 
-static void tick_handler(struct tm *tick_time, TimeUnits changed) {
+static void prv_tick_handler(struct tm *tick_time, TimeUnits changed) {
   // Store time
   s_last_time.hours = tick_time->tm_hour;
   s_last_time.hours -= (s_last_time.hours > 12) ? 12 : 0;
   s_last_time.minutes = tick_time->tm_min;
 
-  for(int i = 0; i < 3; i++) {
+  for (int i = 0; i < 3; i++) {
     s_color_channels[i] = rand() % 256;
   }
 
   // Redraw
-  if(s_canvas_layer) {
+  if (s_canvas_layer) {
     layer_mark_dirty(s_canvas_layer);
   }
 }
 
-static int hours_to_minutes(int hours_out_of_12) {
+static int prv_hours_to_minutes(int hours_out_of_12) {
   return (int)(float)(((float)hours_out_of_12 / 12.0F) * 60.0F);
 }
 
-static void update_proc(Layer *layer, GContext *ctx) {
+static void prv_update_proc(Layer *layer, GContext *ctx) {
+  GRect bounds = layer_get_unobstructed_bounds(layer);
```

Comment:
We should also probably be grabbing the full window bounds (to use when filling the background color)